src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -2185,6 +2185,13 @@ private void ShowOpenDialogAndOpenResult(object parameter)
             }
         }
 
+        private async void ShowOpenDialogAndOpenResultAsync(object parameter)
+        {
+            await Task.Yield();
+
+            ShowOpenDialogAndOpenResult(parameter);
```

Comment:
If this works, then probably using the dispatcher will do the same thing
```
UIDispatcher.BeginInvoke(DispatcherPriority.ApplicationIdle, () => ShowOpenDialogAndOpenResult(parameter))
```

Reply:
Updating this call to use a Dispatcher
see commit: https://github.com/DynamoDS/Dynamo/pull/15361/commits/b88d71d409acfefe18954753d57e545c14cc3191